includes/models/class-nps-survey.php

```diff
@@ -49,6 +49,14 @@ class Nps_Survey {
 	 */
 	private $feedback_question = '';
 
+	/**
+	 * Permalink URL where the NPS is published.
+	 *
+	 * @since [next-version-number]
```

Comment:
So that's 1.4? I like the idea with the placeholder but should we maybe document that somewhere to bulk-update these before a release? (That'd be out of scope for this PR).

Reply:
The `prepare-release.sh` script takes care of it.